process/factory/interceptorscontainer/baseInterceptorsContainerFactory.go

```diff
@@ -580,7 +329,7 @@ func (icf *interceptorsContainerFactory) createOneMiniBlocksInterceptor(topic st
 
 //------- MetachainHeader interceptors
 
-func (icf *interceptorsContainerFactory) generateMetachainHeaderInterceptor() ([]string, []process.Interceptor, error) {
+func (icf *baseInterceptorsContainerFactory) generateMetachainHeaderInterceptor() ([]string, []process.Interceptor, error) {
```

Comment:
be consistent on functions names....generateMetachainHeaderInterceptor and generateHdrInterceptor...use Hdr or Header not both

Reply:
done